code/modules/mob/living/silicon/mommi/mommi.dm

```diff
@@ -85,6 +85,10 @@ They can only use one tool at a time, they can't choose modules, and they have 1
 	//If KEEPER is enabled, disable it.
 	if(keeper)
 		keeper = FALSE
+	
+	//and maybe some more freedoms like staying up past bedtime and cutting in lines :)
+	if(!(quirk_flags & MODULE_CAN_HANDLE_FOOD))
```

Comment:
if(!HAS_MODULE_QUIRK(src, MODULE_CAN_HANDLE_FOOD))

Reply:
That is a lot prettier than what I was doing to fix this, thank you!